include/chimera/visitor.h

```diff
@@ -15,7 +16,7 @@ class Visitor : public clang::RecursiveASTVisitor<Visitor>
 {
 public:
     Visitor(clang::CompilerInstance *ci,
-            std::unique_ptr<CompiledConfiguration> cc);
+            std::shared_ptr<CompiledConfiguration> cc);
```

Comment:
Since we no longer need ownership at all and know the configuration will stay in scope, can we pass by reference here?
```suggestion
            CompiledConfiguration &cc);
```

Reply:
This decision depends on the first comment.